api.planx.uk/send/email.ts

```diff
@@ -125,19 +124,25 @@ const downloadApplicationFiles = async(req: Request, res: Response, next: NextFu
       if (geojson) {
         const geoBuff = Buffer.from(JSON.stringify(geojson, null, 2));
         zip.addFile("boundary.geojson", geoBuff);
+
+        // generate and add an HTML boundary document
+        const boundaryPath = path.join(tmpDir, "boundary.html");
+        const boundaryFile = fs.createWriteStream(boundaryPath);
+        const boundaryStream =
+          generateHTMLMapStream(geojson).pipe(boundaryFile);
+        await resolveStream(boundaryStream);
+        zip.addLocalFile(boundaryPath);
+        deleteFile(boundaryPath);
```

Comment:
adding map file, but waiting on doc templates seems right to me since we know Send to Email is being mostly used for Prior Approval testing right now, where these docs don't apply yet :+1: